examples/lines_bars_and_markers/bar_stacked.py

```diff
@@ -3,30 +3,30 @@
 Stacked bar chart
 =================
 
-This is an example of creating a stacked bar plot with error bars
-using `~matplotlib.pyplot.bar`.  Note the parameters *yerr* used for
-error bars, and *bottom* to stack the coffee's bars on top of the tea's
-bars.
+This is an example of creating a stacked bar plot
+using `~matplotlib.pyplot.bar`.
 """
 
 import matplotlib.pyplot as plt
+import numpy as np
 
+# data from https://allisonhorst.github.io/palmerpenguins/
 
-labels = ['G1', 'G2', 'G3', 'G4', 'G5']
-tea_means = [20, 35, 30, 35, 27]
-coffee_means = [25, 32, 34, 20, 25]
-tea_std = [2, 3, 4, 1, 2]
-coffee_std = [3, 5, 5, 3, 3]
-width = 0.25       # the width of the bars: can also be len(x) sequence
+species = ("Adelie", "Gentoo", "Chinstrap")
+booleans = ["True", "False"]
```

Comment:
Yeah sure we can change the labels to above and below, I like that idea. I think we can try adding the average weights in the labels. 

I don't think it'll work in the title as I've compared the average weight _species wise_. So I'd have to add each average per species. I think in the label makes sense then?

Reply:
![Figure_1](https://user-images.githubusercontent.com/73378227/203860195-f7ae3660-ba19-4fef-ab23-f798f36abec8.png)

What it would look like with the average in labels, could add $\mu = 5076.02g$ for example, to make it even clearer?

Reply:
This is what it might look like:

![Figure_1](https://user-images.githubusercontent.com/73378227/203945519-85efbcf1-91d9-4b53-817d-c6e4f94bd46f.png)

Reply:
Sure. I think they are a little hard to read in the middle of the bars. Mockup below:

![Figure_1](https://user-images.githubusercontent.com/73378227/204054698-dee8ba85-3b0e-4af6-9033-c3e8dc5d7363.png)

Reply:
@story645

Reply:
No worries! Done.